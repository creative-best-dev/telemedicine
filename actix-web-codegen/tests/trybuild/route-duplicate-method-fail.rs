use actix_web::*;

#[route("/", method="GET", method="GET")]
async fn index() -> impl Responder {
    HttpResponse::Ok()
}

#[actix_web::main]
async fn main() {
    let srv = test::start(|| App::new().service(index));

    let request = srv.get("/");
    let response = request.send().await.unwrap();
    assert!(response.status().is_success());
}
