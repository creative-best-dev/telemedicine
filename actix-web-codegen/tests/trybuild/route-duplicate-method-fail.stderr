error: HTTP Method defined more than once: `GET`
 --> $DIR/route-duplicate-method-fail.rs:3:35
  |
3 | #[route("/", method="GET", method="GET")]
  |                                   ^^^^^

error[E0425]: cannot find value `index` in this scope
  --> $DIR/route-duplicate-method-fail.rs:10:49
   |
10 |     let srv = test::start(|| App::new().service(index));
   |                                                 ^^^^^ not found in this scope
